src/Fixer/Comment/AlignMultilineCommentFixer.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Comment;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class AlignMultilineCommentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Multiline comments and docblocks MUST be aligned with comment opening.',
+            [new CodeSample('<?php
+    /*
+     * Multiline comment
+  *
+ Lines not prefixed with asterisk are left untouched
+       *
+   */')]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound([T_COMMENT, T_DOC_COMMENT]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $intendationChars = 0;
+        foreach ($tokens as $index => $token) {
+            $newlinePosition = mb_strrpos($token->getContent(), "\n");
+
+            if ($token->isComment() && '/*' === substr($token->getContent(), 0, 2) && false !== $newlinePosition) {
+                $content = $token->getContent();
+                $content = preg_replace('/^([ \t]+)?\*/m', str_repeat(' ', $intendationChars).'*', $content);
```

Comment:
this is wrong. you need to insert whitespace type requested by user, thus tabs or spaces